### DISCUSSION TITLE
Make an entire directory accessible

### DISCUSSION THREAD

**Discussion by TheTechRobo** [pallets/flask#4177] [Q&A] [answered]
We have an old static version of our site, we don't have the time to update it all to the dynamic version i.e. app.route(), fixing stylesheet links to url_for, etc.

Is there a way to just treat a subdirectory as static HTML, i.e. not even having to use url_for for stylesheets?

**Comment by ThiefMaster** [comment, marked as answer]
You don't need flask for this. Just configure an alias on your webserver so those requests don't even hit your Flask app.

**Reply by TheTechRobo** [reply]
Our entire webserver uses Flask except for  a few things, while we could setup a virtualhost in apache, we want to know if it's possible to just use Flask for this.

**Reply by ThiefMaster** [reply]
Flask is not a webserver, your Flask app should run behind a real webserver (like Apache or nginx) anyway. And those web servers are MUCH BETTER at service purely static content - so defining an `Alias /foo /path/to/foo/data` (no need for a full-blown virtualhost) is the correct solution.

That said, nothing stops you from creating an endpoint path a path-type argument (accepts slashes), and then use `send_from_directory` to serve that path from a given directory... It's just not efficient compared to letting the web server do it.

**Reply by TheTechRobo** [reply]
Okay, thanks for the answer!

(By the way - I know that flask is not a webserver, I meant our entire webSITE)
